packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -8,13 +8,12 @@ import '@vaadin/button/src/vaadin-button.js';
 import './vaadin-month-calendar.js';
```

Comment:
The focused date element has some kind of a focus ring as you scroll down and the same calendar instance is re-used

https://user-images.githubusercontent.com/1222264/152347352-b16cfbab-ebd8-42ab-8ece-be874604d6d1.mp4

Reply:
Good catch. The outline was due to native `:focus` state and it was only reset by `[focused]` selector.
When scrolling, the `focused` attribute was removed which caused this. Fixed.